lib/Util/SVFModule.cpp

```diff
@@ -36,11 +36,34 @@
 #include <llvm/IRReader/IRReader.h>	// IR reader for bit file
 #include <llvm/Support/FileSystem.h>	// for sys::fs::F_None
 
+#include <llvm/IR/IRBuilder.h>
+/*
+
+  svf.main() is used to model the real entry point of a C++ program,
+  which initializes all global C++ objects and then call main().
+  For example, given a "int main(int argc, char * argv[])", the corresponding
+  svf.main will be generated as follows:
+
+    define void @svf.main(i32, i8**, i8**) {
+      entry:
+        call void @_GLOBAL__sub_I_cast.cpp()
+        call void @_GLOBAL__sub_I_1.cpp()
+        call void @_GLOBAL__sub_I_2.cpp()
+        %3 = call i32 @main(i32 %0, i8** %1)
+        ret void
+    }
+ */
+#define SVF_MAIN_FUNC_NAME           "svf.main"
+#define SVF_GLOBAL_SUB_I_XXX          "_GLOBAL__sub_I_"
+
 using namespace std;
 using namespace llvm;
 
 static cl::opt<std::string> Graphtxt("graphtxt", cl::value_desc("filename"),
                                      cl::desc("graph txt file to build PAG"));
+                                     
+static cl::opt<bool> SVFMain("svfmain", cl::init(false),
```

Comment:
Shall we turn it on by default? please run few tests to make sure it is correct.